subsys/caf/modules/Kconfig.ble_adv

```diff
@@ -8,6 +8,7 @@ menuconfig CAF_BLE_ADV
 	bool "Enable Bluetooth LE advertising"
 	depends on BT_PERIPHERAL
 	depends on CAF_BLE_COMMON_EVENTS
+	depends on CAF_BLE_STATE
```

Comment:
This line seems like a fix for the dependency description. If so, please move it to a separate commit